src/main.ts

```diff
@@ -388,13 +391,21 @@ async function main() {
             .status(400)
             .send({ error: 'Invalid request body', contract, chainNonce });
         }
-        console.log({ config_scan });
+        const chainConfig = getChain(String(chainNonce));
```

Comment:
the response from getChain can be of type ChainConfig | undefined. Add checks if it does not find the chain